//Runtime Error
public class ExceptionThird {
    public static void main(String[] args) 
    {
        int num1 = 6;
        int num2 = 0;
        int result = num1/num2;
        System.out.println(result);

    }
}
